.expeditor/verify.pipeline.yml

```diff
@@ -44,7 +44,27 @@ steps:
   expeditor:
     executor:
       docker:
-        image: ruby:2.7-buster
+        image: ruby:2.7
+
+- label: run-lint-and-specs-ruby-3.0
+  command:
+    - apt-get update
+    - apt-get install -y libarchive-dev
```

Comment:
Is there a reason to install libarchive here?

Reply:
Due to conflict in branch name used. I renamed the branch and opened  new pR (https://github.com/chef/mixlib-versioning/pull/52) for the same .
Could you please Review the new PR.